apps/docs/app/global.css

```
@@ -2,31 +2,769 @@
+html {
+  min-width: 320px;
+  scroll-behavior: smooth;
+  background: var(--night-ink);
+}
+
+body {
+  margin: 0;
+  background: var(--night-ink);
+  color: var(--signal-white);
+  font-family: var(--font-geist-sans), Arial, sans-serif;
+  text-rendering: optimizeLegibility;
```

Stylelint (17.14.0)

[error] Expected "optimizeLegibility" to be "optimizelegibility" (value-keyword-case)

(value-keyword-case)